src/sentry/options/defaults.py

```diff
@@ -362,6 +362,9 @@
 # All Relay options (statically authenticated Relays can be registered here)
 register("relay.static_auth", default={}, flags=FLAG_NOSTORE)
 
+# Tell Relay to stop extracting metrics from transaction payloads
+register("relay.drop-transaction-metrics", default=[])
```

Comment:
Could you add a simple description of what to put into this list?

Reply:
As a nitpick, from https://github.com/getsentry/sentry/pull/36188 it seems that this kill switch only allows to specify projects. Am I reading this right? If so, we should add this into the name of the option to make it more clear.

Reply:
Good point. Do we want another kill switch that disables extraction for all projects? If so, do we schedule invalidation tasks or not?

Reply:
to summarize from standup, one can configure this killswitch with `project_id: null` and it will match all projects

Reply:
It seems like `project_id: null` does not work at the moment. But once we have fixed that, I will add it as an example in the comment.